### PR TITLE
Update phylo tree/run models

### PR DESCRIPTION
**Pull request by ttung**
### Description
1. Add a relationship to express which samples were actually incorporated into a tree.
2. Add data about how the build is made.

Depends on #230 

#### Issue
[ch129581](https://app.clubhouse.io/genepi/story/129581/schema-changes-to-record-which-samples-are-in-a-tree)
[ch130359](https://app.clubhouse.io/genepi/story/130359/schema-changes-to-record-the-build-configuration)

### Test plan
reran the import and pushed the docker image to ECR.

### CODE REVIEW

**Review by shanaxel42 — APPROVED** [review]
lgtm

File: src/py/aspen/database/models/phylo_tree.py [hunk 1]
```diff
@@ -1,13 +1,40 @@
...
+    base.metadata,
+    Column("sample_id", ForeignKey(Sample.id), primary_key=True),
+    Column(
+        "phylo_tree_id",
+        ForeignKey(f"{_PHYLO_TREE_TABLENAME}.entity_id"),
+        primary_key=True,
+    ),
+)
+"""This table records which samples are included in the phylo tree.  Note that this is
+different from which samples are inputs into the build that creates a phylo tree."""
```
Comment (phoenixAja) [inline src/py/aspen/database/models/phylo_tree.py:33]: can we add this as a docstring to whatever model this is referring to?

File: src/py/aspen/database/models/phylo_tree.py [hunk 2]
```diff
@@ -29,6 +63,19 @@ class PhyloRun(Workflow):
     )
     group = relationship(Group, backref=backref("phylo_runs", uselist=True))
 
+    template_file_path = Column(String, nullable=True)
+    """This is the path, relative to aspen root, for the builds template file.  For all
+    new builds, this should be set.  However, for historically imported """
```
Comment (phoenixAja) [inline src/py/aspen/database/models/phylo_tree.py]: this sentence looks like it was cutoff

**Review by phoenixAja — APPROVED** [review]

File: src/py/aspen/database/models/phylo_tree.py (hunk 1 above)
Comment (ttung) [reply src/py/aspen/database/models/phylo_tree.py:33]: This is applicable to `PhyloTreeSamples`.  It is somewhat a quirk of python docs that the docstring appears below, even in the case of variables.

File: src/py/aspen/database/models/phylo_tree.py (hunk 2 above)
Comment (ttung) [reply src/py/aspen/database/models/phylo_tree.py]: HAHA thanks for noticing this.

Comment (ttung) [reply src/py/aspen/database/models/phylo_tree.py]: and fixed.

**Pull request merged**